### PR TITLE
added the LiteralSource value store

### PR DESCRIPTION
**Pull request by daniel-shields**

### CODE REVIEW

File: src/uberjob/stores/_literal_source.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    def get_modified_time(self) -> typing.Optional[dt.datetime]:
+        """Get the modified time."""
+        return self.modified_time
+
+    def __repr__(self):
+        return repr_helper(
+            self,
+            self.value,
+            self.modified_time.isoformat() if self.modified_time else None,
```
Comment (daniel-shields) [inline src/uberjob/stores/_literal_source.py:52]: ModifiedTimeSource also uses isoformat(), and it's not clear whether this is better than repr(), which seems more correct.

File: src/uberjob/stores/_literal_source.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    A :class:`~uberjob.ValueStore` that takes value and modified time in its constructor
+    and simply returns them from ``read`` and ``get_modified_time``.
+
+    :param value: The value.
+    :param modified_time: The modified time.
+    """
+
+    def __init__(self, value, modified_time: typing.Optional[dt.datetime]):
+        self.value = value
+        self.modified_time = modified_time
```
Comment (daniel-shields) [inline src/uberjob/stores/_literal_source.py:34]: ModifiedTimeSource checks that this is either None or a dt.datetime, but type checking elsewhere in the value stores is not really a thing. Maybe the type check should be removed from ModifiedTimeSource?

Comment (timothy-shields) [reply src/uberjob/stores/_literal_source.py:34]: I think I lean the other way. We should probably be doing validation in these other value stores.
Let's not deal with that in this set of changes.

File: src/uberjob/stores/_literal_source.py (hunk 1 above)
Comment (timothy-shields) [reply src/uberjob/stores/_literal_source.py:52]: Agreed, let's change these separately.

**Review by timothy-shields — APPROVED** [review]

**Review by timothy-shields — APPROVED** [review]

**Pull request merged**